### PR TITLE
feat: external news on blogs page

### PR DESCRIPTION
**Pull request by mageexcel**

### CODE REVIEW

**Comment by Skraye** [comment]
All images should have the same size :
image
image

File: pages/blogs/[...slug].vue [hunk 1]
```diff
@@ -1,6 +1,18 @@
 <template>
     <div class="container">
-        <BlogsList v-if="slug === '/blogs/'" :blogs="page"/>
+        <BlogsList v-if="slug === '/blogs/'" :blogs="page" :external-news="externalNews"/>
+
+        <div v-else-if="slug === '/blogs/community'">
+            <div class="mt-5 row">
+                <DocsBreadcrumb :slug="slug" :page-list="[]"></DocsBreadcrumb>
```
Comment (Skraye) [inline pages/blogs/[...slug].vue:7]: add the blog page in the page-list so you can go back to blog list by clicking on it

**Comment by Skraye** [comment]
Blog cannot be accessed anymore (the kestra one, not community)

**Comment by tchiotludo** [comment]
for the image size, you should leverage on [nuxt/image](https://image.nuxt.com/) that we have already installed, just whitelist domain from google that we will always use

**Pull request merged**